.github/workflows/documentation.yml

```diff
@@ -8,7 +8,10 @@ on:
       - '*.md'
       - '.github/workflows/documentation.yml'
 
-permissions: {}
+permissions: 
```

Comment:
This can be set to `permissions: {}` with the required block set down in the job that requires them

Reply:
set back to original